Regression/Checksum/benchmarks_json/Langmuir_multi_psatd_single_precision.json

```
@@ -2,31 +2,31 @@
   "electrons": {
     "particle_cpu": 131072.0,
     "particle_id": 18270519296.0,
-    "particle_momentum_x": 9.638206108462784e-20,
-    "particle_position_x": 2.621439946144335,
-    "particle_position_y": 2.6214399605720473,
-    "particle_position_z": 2.6214399608165024,
+    "particle_momentum_x": 9.638045709465868e-20,
+    "particle_position_x": 2.62143994388461,
+    "particle_position_y": 2.621439958560387,
+    "particle_position_z": 2.6214399584409023,
     "particle_weight": 127999991808.0
   },
   "lev=0": {
-    "Bx": 11.942729428141796,
-    "By": 11.947761789089679,
-    "Bz": 11.942489514683562,
-    "Ex": 84780606024507.0,
-    "Ey": 84780616397484.88,
-    "Ez": 84780559670537.38,
-    "jx": 6.087564613231104e+16,
-    "jy": 6.087565743692762e+16,
-    "jz": 6.087567343856218e+16,
+    "Bx": 11.95347876738498,
+    "By": 11.956506397748562,
+    "Bz": 11.952493572239355,
+    "Ex": 84779212317896.12,
+    "Ey": 84779216019354.88,
+    "Ez": 84779165259258.0,
+    "jx": 6.087463260557299e+16,
+    "jy": 6.087464639979942e+16,
+    "jz": 6.087466379301542e+16,
     "part_per_cell": 524288.0,
-    "rho": 702987811.1679688
+    "rho": 702976367.7089844
   },
   "positrons": {
     "particle_cpu": 131072.0,
     "particle_id": 54811295744.0,
-    "particle_momentum_z": 9.638209718562555e-20,
-    "particle_position_x": 2.6214399461232745,
-    "particle_position_y": 2.621439960570399,
-    "particle_position_z": 2.621439960792145
+    "particle_momentum_z": 9.638049808110674e-20,
+    "particle_position_x": 2.6214399436821623,
+    "particle_position_y": 2.621439958363709,
+    "particle_position_z": 2.6214399583751913
   }
 }
```

```suggestion
}

```
Also missing a new line